persistence/provider/sql/postgres/error.go

```diff
@@ -170,6 +171,56 @@ func (d errorConverter) ScanEvent(
 	return d.d.ScanEvent(rows, i)
 }
 
+//
```

Comment:
Shouldn't these doc comments be on the functions?

Reply:
The

```
//
// offsetstore
//
``` 

is just a separator between methods from different interfaces, but the `// LoadOffset` docs can be omitted entirely from this file.

Reply:
For clarity: I leave the docs out of this file because it's just another place to keep them in sync, and this file doesn't actually implement any of the logic described in the docs, it's just a passthrough that fixes some bung errors that the postgresql driver returns.